packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddAccountModal/AddAccountModal.tsx

```diff
@@ -27,6 +27,7 @@ import { DeviceModelInternal } from '@trezor/connect';
 
 const StyledModal = styled(Modal)`
     width: 480px;
+    padding-top: 4px;
```

Comment:
Not sure if this PR is still relevant, but we prefer using `spacingsPx` instead, if possible. So in your case, it would look like this:

`padding-top: ${spacingsPx.xxs}; `

Anyway, if we still needed to add padding to this modal, maybe I'd add a bit bigger padding, 4px seem to be too little. `spacingsPx`.xs could look better, IMHO :)